api/mesh/v1alpha1/mesh.proto

```diff
@@ -199,6 +199,10 @@ message DatadogTracingBackendConfig {
 
   // Port of datadog collector
   uint32 port = 2 [ (doc.required) = true ];
+
+  // Determines if datadog service name should be split based on traffic
+  // direction and destination. Default: false
```

Comment:
How about we include an example of what the names look like if this is true?

Reply:
Like the ones in https://github.com/kumahq/kuma-website/pull/1012.

Reply:
updated